OKR.md

```diff
@@ -0,0 +1,39 @@
+# Nodejs.dev Objectives and Key Results
+
+The Nodejs.dev project uses Objectives and Key Results to try to provide direction and guidance to the work we to, enabling our own work and the work of our initiatives to have a north star for us to ensure we're working towards.
```

Comment:
```suggestion
The Nodejs.dev project uses Objectives and Key Results to provide direction and guidance to the work we do, enabling our own work and the work of our initiatives to have a north star for us to work towards.
```

Initially I was going to just fix a typo `s/to/do` but I think we can also make this a little clearer/shorter too. Feel free to ignore these suggestions as long as the "the work we to" is changed to "the work we do". :)